paulshryock/paul-shryock
# [#12/#13]: Transpile and polyfill modern JavaScript to ES5.

## Summary
Modern JavaScript is already transpiled to ES2015, which covers [94% of all users](https://caniuse.com/es6). This is served to supporting browsers via `type=module`.

For those with unsupporting browsers, ES2015 is further transpiled and polyfilled to ES5 in a `.legacy.js` bundle. This is served to legacy browsers via `nomodule`.

## Testing
- Run `npm start`
- Confirm that modern JavaScript is transpiled and polyfilled to ES5 in modern and legacy bundles.

## Issue(s)

Closes #12 and #13

## Changelog

### Added
- Transpile modern JavaScript to ES5. [#12]
- Polyfill modern JavaScript to ES5. [#13]

## Release Checklist
